fenix/app/src/main/res/values/strings.xml

```diff
@@ -317,9 +317,13 @@
     <!-- Text for the button to set firefox as default browser on the device -->
     <!-- Title for set firefox as default browser screen used by Nimbus experiments. -->
     <string name="juno_onboarding_default_browser_title_nimbus_2">We love keeping you safe</string>
+    <!-- Title for set firefox as default browser screen used by Nimbus experiments. -->
+    <string name="juno_onboarding_default_browser_title_nimbus_4" tools:ignore="UnusedResources">Safe browsing with more choices</string>
     <!-- Description for set firefox as default browser screen used by Nimbus experiments. -->
     <string name="juno_onboarding_default_browser_description_nimbus_3">Our non-profit backed browser helps stop companies from secretly following you around the web.</string>
     <!-- Description for set firefox as default browser screen used by Nimbus experiments. -->
+    <string name="juno_onboarding_default_browser_description_nimbus_5" tools:ignore="UnusedResources">Known trackers? Blocked automatically. Extensions? Try all 600. PDF’s? Our built-in reader makes them easy to manage.</string>
```

Comment:
There are currently 766 add-ons available: https://addons.mozilla.org/en-US/android/search/
Yet this string calls out 600.
We might want to double check if 600 is really the number we want to call out here

Reply:
@Delphine thanks for asking this 🙂 per the latest comms this should be 700

Reply:
It looks like there's a typo as well. PDF's should be PDFs. Thanks!

Reply:
Thanks @ewachowiak have updated that copy